packages/flutter/lib/src/material/button_style.dart

```diff
@@ -65,8 +65,8 @@ import 'theme_data.dart';
 /// useful to make relatively sweeping changes based on a few initial
 /// parameters with simple values. The button styleFrom() methods
 /// enable such sweeping changes. See for example:
-/// [TextButton.styleFrom], [ElevatedButton.styleFrom],
-/// [OutlinedButton.styleFrom].
+/// [ElevatedButton.styleFrom], [FilledButton.styleFrom],
```

Comment:
Is there somewhere in docs that explains how to achieve the tonal look with `FilledButton.styleFrom`?

Reply:
No. We used to have an example that showed how to do it with and ElevatedButton, but I updated it to just use FilledButton.tonal.

Reply:
Gotcha. I commented where I think docs would benefit from a small example.